website/common/script/libs/shops.js

```diff
@@ -114,6 +114,27 @@ shops.getMarketCategories = function getMarket (user, language) {
 shops.getQuestShopCategories = function getQuestShopCategories (user, language) {
   let categories = [];
 
+  let bundleCategory = {
+    identifier: 'bundle',
+    text: i18n.t('questBundles', language),
+  };
+
+  bundleCategory.items = sortBy(values(content.bundles)
```

Comment:
Ah, OK, I slightly misunderstood what you were talking about. I'll see what I can come up with!